WordPress/gutenberg
# Fix wp-scripts watch mode failing when `block.json` contains malformed JSON

## What?

Fixing the watch mode of `@wordpress/scripts` to no longer abort when it encounters malformed JSON.

## Why?

Currently the `watch` mode of the `@wordpress/scripts` package fails whenever you either add a new empty `block.json` file, or save it with malformed JSON in it. 

## How?

Wrapping the `JSON.parse` function call in a `try catch` block

## Testing Instructions

1. Start the watch mode of `@wordpress/scripts` in a project
2. Add a new empty block.json file or save a block.json file with some malformed json in it
3. See that it no longer aborts the watch mode
